runtime/default.policy

```diff
@@ -104,6 +104,8 @@ rules:
           (chmod.file.path in [ "/etc/shadow", "/etc/gshadow" ])
           && process.file.path not in [ "/sbin/vipw", "/usr/sbin/vipw", "/sbin/vigr", "/usr/sbin/vigr" ]
           && process.ancestors.file.path not in SYSTEM_PACKAGE_MANAGERS
+          && (process.file.name not in ["containerd"] && process.args in ["info"])
```

Comment:
```suggestion
          && !(process.file.name in ["containerd"] && process.args in ["info"])
```

Reply:
I think the boolean logic is wrong here (and in all similar rules). Did this rule work when you tested it locally ?

Reply:
You're right. Fixed in https://github.com/DataDog/security-monitoring/pull/938